occurrences/models.py

```diff
@@ -224,6 +225,15 @@ class Enrolment(models.Model):
         (NOTIFICATION_TYPE_EMAIL, _("email")),
         (NOTIFICATION_TYPE_SMS, _("sms")),
     )
+    STATUS_APPROVED = "approved"
+    STATUS_PENDING = "pending"
+    STATUS_CANCELLED = "cancelled"
+    STATUSES = (
+        (STATUS_APPROVED, _("approved")),
```

Comment:
Maybe we should have also status for declining enrolments. The use case could be when the provider declines the enrolment for some reason.

Reply:
Added `Declined` status and mutaiton to decline enrolment